sr_demos/scripts/sr_demos/close.py

```diff
@@ -76,11 +76,5 @@
         }
     ]
 
-    if hand_name == "two_hands":
-        lh_commander_instance = SrHandCommander("left_hand")
-        rh_commander_instance = SrHandCommander("right_hand")
-        lh_commander_instance.run_named_trajectory(trajectory)
-        rh_commander_instance.run_named_trajectory(trajectory)
-    else:
-        hand_commander_instance = SrHandCommander(name=hand_name)
-        hand_commander_instance.run_named_trajectory(trajectory)
+    commander_instance = SrHandCommander(hand_name)
```

Comment:
It's worth removing all the `joint_prefix_name` stuff from further up, since it's not used for anything.

Reply:
sorry about that. fixed in 4b2d32fdb479c15c8e3e04278ba59fd405ef4f04